compiler/damlc/tests/daml-test-files/RecordUpdateExp.daml

```diff
@@ -0,0 +1,16 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+daml 1.2
+module RecordUpdateExp where
+
+data A = A with x: Int
+data B = B with y: A; z: A
+
+-- Check parsing/linting.
+
+f : Int -> A -> B
+f x a = B a with
```

Comment:
Is there a way to put this onto one line without getting the "redundant brackets" lint? If not, we should disable the lint by default.

Reply:
Oops that was my mistake mucking around. Let me fix that.

Reply:
Fixed. As I mentioned earlier `f x a = B a with x a with x` is perfectly valid.